.github/workflows/continuous-integration.yml

```diff
@@ -57,45 +57,6 @@ jobs:
           report_paths: 'test-results/unit-tests.xml'
           summary: ${{ steps.code-coverage.outputs.MARKDOWN }}
 
-  linting:
```

Comment:
I think we'll still want the linting to happen in the main CI workflow. If there's an issue, it should be caught and fixed before getting archived, deployed, etc.

Reply:
We might want linting both for PRs and in the main CI (but only in master to avoid redundancy).

I just want to avoid the case where the combination of two different commits merged to master causes us to miss a linting issue.

We could probably set this condition on the linting job in the CI workflow.
```yml
if: github.ref == 'refs/heads/master'
```

Reply:
Added conditional statement to check across all files for master not just committed